frontend/src/core/services/auth.service.js

```diff
@@ -1,6 +1,8 @@
 import { http } from "../utils";
 
 const API_URL = process.env.NEXT_PUBLIC_MOONSTREAM_API_URL;
+const DB_URL = process.env.NEXT_PUBLIC_MOONSTREAM_DB_URL;
```

Comment:
This isn't in the check at the top level. Is it already defined?

Reply:
Not defined anywhere, I added this in `sample.env`